### ISSUE TITLE
nwjs 0.13 beta3 - devtools / app crash when clicking links from console

### ISSUE THREAD

**Issue by AndryBray** [nwjs/nw.js#4256]
Hello,
I found a problem when I try to click some link from devtools console (for example debug link from google maps or some warning inside my code)

It's enough to create e new page including the google maps api without API key or setting a bas params, you will see a warning inside the console, click on this link and the app will crash.

```
<script type='text/javascript' src='https://maps.googleapis.com/maps/api/js?v=3&sensor=false></script>
```
MacOS Report of crash here:
https://gist.github.com/AndryBray/a6ce61eeddb8ef1aedc0

**Comment by jonwwilkes** [comment]
Can confirm on Linux 32bit

**Comment by offshore** [comment]
**Maybe** related to #4121 (however it's reported fixed in beta3).

**Comment by ghostoy** [comment]
@AndryBray @jonwwilkes @offshore Could you post a full sample whose console log crashes NW?

**Comment by ghostoy** [comment]
![image](https://cloud.githubusercontent.com/assets/471928/12391961/66c5eb80-be26-11e5-8aa6-245ae91ca8cc.png)
Seemed that I didn't wait long to see the log. It's the link in the log message that breaks NW.js, not the links of the source code (which was fixed in #4121).

**Comment by ghostoy** [comment]
Merged to #4269
